src/hotspot/share/opto/node.hpp

```diff
@@ -2000,6 +2000,10 @@ Compile::locate_node_notes(GrowableArray<Node_Notes*>* arr,
   return arr->at(block_idx) + (idx & (_node_notes_block_size-1));
 }
 
+inline Node_Notes* Compile::node_notes_at(int idx) {
```

Comment:
It seems weird to have these inlined Compile methods in node.hpp.  Could we clean this up separately, maybe by moving them to compile.inline.hpp?